{"uid":"3aea7ea94d9c00c8","name":"Logout off the site succesfully","fullName":"Logout off the site succesfully","historyId":"Logout off the site succesfully","time":{"start":1677698118366,"stop":1677698118582,"duration":216},"status":"broken","statusMessage":"An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:4444/session. The exception message was: An error occurred while sending the request. -> An error occurred while sending the request. -> Unable to read data from the transport connection: Connection reset by peer. -> Connection reset by peer","statusTrace":"OpenQA.Selenium.WebDriverException: An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:4444/session. The exception message was: An error occurred while sending the request.\n ---> System.Net.Http.HttpRequestException: An error occurred while sending the request.\n ---> System.IO.IOException: Unable to read data from the transport connection: Connection reset by peer.\n ---> System.Net.Sockets.SocketException (104): Connection reset by peer\n   --- End of inner exception stack trace ---\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource<System.Int32>.GetResult(Int16 token)\n   at System.Net.Http.HttpConnection.InitialFillAsync(Boolean async)\n   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   --- End of inner exception stack trace ---\n   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.SendWithVersionDetectionAndRetryAsync(HttpRequestMessage request, Boolean async, Boolean doRequestAuth, CancellationToken cancellationToken)\n   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpClient.<SendAsync>g__Core|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.MakeHttpRequest(HttpRequestInfo requestInfo)\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.Execute(Command commandToExecute)\n   --- End of inner exception stack trace ---\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.Execute(Command commandToExecute)\n   at OpenQA.Selenium.WebDriver.Execute(String driverCommandToExecute, Dictionary`2 parameters)\n   at OpenQA.Selenium.WebDriver.StartSession(ICapabilities desiredCapabilities)\n   at OpenQA.Selenium.WebDriver..ctor(ICommandExecutor executor, ICapabilities capabilities)\n   at OpenQA.Selenium.Remote.RemoteWebDriver..ctor(ICommandExecutor commandExecutor, ICapabilities desiredCapabilities)\n   at OpenQA.Selenium.Remote.RemoteWebDriver..ctor(Uri remoteAddress, ICapabilities desiredCapabilities, TimeSpan commandTimeout)\n   at OpenQA.Selenium.Remote.RemoteWebDriver..ctor(Uri remoteAddress, ICapabilities desiredCapabilities)\n   at OpenQA.Selenium.Remote.RemoteWebDriver..ctor(Uri remoteAddress, DriverOptions options)\n   at SeleniumTest.Core.Drivers.RemoteWebDriverFactory.RemoteConfigs(DriverOptions capabilities) in /home/sch/actions-runner/_work/GUI-Framework/GUI-Framework/1.Core/UIElements/Drivers/RemoteWebDriver.cs:line 33\n   at SeleniumTest.Core.Drivers.RemoteWebDriverFactory.GetDriver(String driverType) in /home/sch/actions-runner/_work/GUI-Framework/GUI-Framework/1.Core/UIElements/Drivers/RemoteWebDriver.cs:line 19\n   at Todoly.Core.UIElements.Drivers.GenericWebDriver.LoadDriver() in /home/sch/actions-runner/_work/GUI-Framework/GUI-Framework/1.Core/UIElements/Drivers/GenericWebDriver.cs:line 28\n   at Todoly.Core.UIElements.Drivers.GenericWebDriver.get_Instance() in /home/sch/actions-runner/_work/GUI-Framework/GUI-Framework/1.Core/UIElements/Drivers/GenericWebDriver.cs:line 11\n   at SeleniumTest.Tests.Hooks.Hooks.CaptureScreenshot() in /home/sch/actions-runner/_work/GUI-Framework/GUI-Framework/3.Tests/UI/Hooks/Hooks.cs:line 161\n   at TechTalk.SpecFlow.Bindings.BindingInvoker.InvokeBinding(IBinding binding, IContextManager contextManager, Object[] arguments, ITestTracer testTracer, TimeSpan& duration)\n   at Allure.SpecFlowPlugin.AllureBindingInvoker.InvokeBinding(IBinding binding, IContextManager contextManager, Object[] arguments, ITestTracer testTracer, TimeSpan& duration)","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"BeforeFeature [10000]","time":{"start":1677698117996,"stop":1677698118093,"duration":97},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"CreateProject [1]","time":{"start":1677698118112,"stop":1677698118329,"duration":217},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"CreateItems [2]","time":{"start":1677698118329,"stop":1677698118331,"duration":2},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"BeforeScenario [10000]","time":{"start":1677698118331,"stop":1677698118338,"duration":7},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"status":"broken","statusMessage":"An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:4444/session. The exception message was: An error occurred while sending the request. -> An error occurred while sending the request. -> Unable to read data from the transport connection: Connection reset by peer. -> Connection reset by peer","statusTrace":"OpenQA.Selenium.WebDriverException: An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:4444/session. The exception message was: An error occurred while sending the request.\n ---> System.Net.Http.HttpRequestException: An error occurred while sending the request.\n ---> System.IO.IOException: Unable to read data from the transport connection: Connection reset by peer.\n ---> System.Net.Sockets.SocketException (104): Connection reset by peer\n   --- End of inner exception stack trace ---\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource<System.Int32>.GetResult(Int16 token)\n   at System.Net.Http.HttpConnection.InitialFillAsync(Boolean async)\n   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   --- End of inner exception stack trace ---\n   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.SendWithVersionDetectionAndRetryAsync(HttpRequestMessage request, Boolean async, Boolean doRequestAuth, CancellationToken cancellationToken)\n   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpClient.<SendAsync>g__Core|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.MakeHttpRequest(HttpRequestInfo requestInfo)\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.Execute(Command commandToExecute)\n   --- End of inner exception stack trace ---\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.Execute(Command commandToExecute)\n   at OpenQA.Selenium.WebDriver.Execute(String driverCommandToExecute, Dictionary`2 parameters)\n   at OpenQA.Selenium.WebDriver.StartSession(ICapabilities desiredCapabilities)\n   at OpenQA.Selenium.WebDriver..ctor(ICommandExecutor executor, ICapabilities capabilities)\n   at OpenQA.Selenium.Remote.RemoteWebDriver..ctor(ICommandExecutor commandExecutor, ICapabilities desiredCapabilities)\n   at OpenQA.Selenium.Remote.RemoteWebDriver..ctor(Uri remoteAddress, ICapabilities desiredCapabilities, TimeSpan commandTimeout)\n   at OpenQA.Selenium.Remote.RemoteWebDriver..ctor(Uri remoteAddress, ICapabilities desiredCapabilities)\n   at OpenQA.Selenium.Remote.RemoteWebDriver..ctor(Uri remoteAddress, DriverOptions options)\n   at SeleniumTest.Core.Drivers.RemoteWebDriverFactory.RemoteConfigs(DriverOptions capabilities) in /home/sch/actions-runner/_work/GUI-Framework/GUI-Framework/1.Core/UIElements/Drivers/RemoteWebDriver.cs:line 33\n   at SeleniumTest.Core.Drivers.RemoteWebDriverFactory.GetDriver(String driverType) in /home/sch/actions-runner/_work/GUI-Framework/GUI-Framework/1.Core/UIElements/Drivers/RemoteWebDriver.cs:line 19\n   at Todoly.Core.UIElements.Drivers.GenericWebDriver.LoadDriver() in /home/sch/actions-runner/_work/GUI-Framework/GUI-Framework/1.Core/UIElements/Drivers/GenericWebDriver.cs:line 28\n   at Todoly.Core.UIElements.Drivers.GenericWebDriver.get_Instance() in /home/sch/actions-runner/_work/GUI-Framework/GUI-Framework/1.Core/UIElements/Drivers/GenericWebDriver.cs:line 11\n   at SeleniumTest.Tests.Hooks.Hooks.CaptureScreenshot() in /home/sch/actions-runner/_work/GUI-Framework/GUI-Framework/3.Tests/UI/Hooks/Hooks.cs:line 161\n   at TechTalk.SpecFlow.Bindings.BindingInvoker.InvokeBinding(IBinding binding, IContextManager contextManager, Object[] arguments, ITestTracer testTracer, TimeSpan& duration)\n   at Allure.SpecFlowPlugin.AllureBindingInvoker.InvokeBinding(IBinding binding, IContextManager contextManager, Object[] arguments, ITestTracer testTracer, TimeSpan& duration)","steps":[{"name":"Given  the user is logged in","time":{"start":1677698118393,"stop":1677698118551,"duration":158},"status":"failed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"attachments":[],"parameters":[],"stepsCount":1,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[{"name":"CaptureScreenshot [10000]","time":{"start":1677698118583,"stop":1677698118614,"duration":31},"status":"broken","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"AfterFeature [10000]","time":{"start":1677698118641,"stop":1677698118642,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"labels":[{"name":"thread","value":"NonParallelWorker"},{"name":"host","value":"sch"},{"name":"feature","value":"User"},{"name":"tag","value":"Smoke"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":8,"skipped":0,"passed":4,"unknown":0,"total":12},"items":[{"uid":"94a95bde7fd49203","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/712//#testresult/94a95bde7fd49203","status":"broken","statusDetails":"An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:4444/session. The exception message was: An error occurred while sending the request. -> An error occurred while sending the request. -> The response ended prematurely.","time":{"start":1677697882624,"stop":1677697882857,"duration":233}},{"uid":"be0ab8b1f24fe6c3","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/712//#testresult/be0ab8b1f24fe6c3","status":"broken","statusDetails":"An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:4444/session. The exception message was: An error occurred while sending the request. -> An error occurred while sending the request. -> Unable to read data from the transport connection: Connection reset by peer. -> Connection reset by peer","time":{"start":1677697802809,"stop":1677697802978,"duration":169}},{"uid":"87506427cef8b67e","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/711//#testresult/87506427cef8b67e","status":"broken","statusDetails":"An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:4444/session. The exception message was: An error occurred while sending the request. -> An error occurred while sending the request. -> The response ended prematurely.","time":{"start":1677697661730,"stop":1677697661987,"duration":257}},{"uid":"b6abe43bf33c49f0","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/709//#testresult/b6abe43bf33c49f0","status":"passed","time":{"start":1677697454325,"stop":1677697457976,"duration":3651}},{"uid":"625245bc9521fc2","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/710//#testresult/625245bc9521fc2","status":"broken","statusDetails":"An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:4444/session. The exception message was: An error occurred while sending the request. -> An error occurred while sending the request. -> Unable to read data from the transport connection: Connection reset by peer. -> Connection reset by peer","time":{"start":1677697366356,"stop":1677697366518,"duration":162}},{"uid":"4523363896386a48","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/710//#testresult/4523363896386a48","status":"passed","time":{"start":1677697260130,"stop":1677697271440,"duration":11310}},{"uid":"8c29431ef67eb572","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/706//#testresult/8c29431ef67eb572","status":"broken","statusDetails":"An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:4444/session. The exception message was: An error occurred while sending the request. -> An error occurred while sending the request. -> The response ended prematurely.","time":{"start":1677696840567,"stop":1677696840885,"duration":318}},{"uid":"a75b3ff5cea98af6","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/706//#testresult/a75b3ff5cea98af6","status":"broken","statusDetails":"An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:4444/session. The exception message was: An error occurred while sending the request. -> An error occurred while sending the request. -> The response ended prematurely.","time":{"start":1677696772540,"stop":1677696772824,"duration":284}},{"uid":"552a68c5f97b437","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/706//#testresult/552a68c5f97b437","status":"passed","time":{"start":1677696365377,"stop":1677696375784,"duration":10407}},{"uid":"8c3cf250ac6a1010","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/709//#testresult/8c3cf250ac6a1010","status":"passed","time":{"start":1677696001773,"stop":1677696005648,"duration":3875}},{"uid":"1a13662a07fb0b1b","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/708//#testresult/1a13662a07fb0b1b","status":"broken","statusDetails":"An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:4444/session/a634a70e2171a28dd114a66c03e50b47/screenshot. The exception message was: Connection refused (localhost:4444) -> Connection refused (localhost:4444) -> Connection refused","time":{"start":1677695465138,"stop":1677695465143,"duration":5}}]},"tags":["Smoke"]},"source":"3aea7ea94d9c00c8.json","parameterValues":[]}